packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/02-plugin-types.md

```diff
@@ -1,22 +1,24 @@
 ---
-title: Plugin Types
+title: How to choose the best plugin type
 ---
 
 ## Choosing the Base Contract for Your Plugin
 
-In this section, we will learn about the interfaces or base contracts you can inherit from when developing a plugin (this is, however, not mandatory).
+Although it is not mandatory to choose one of our interfaces as the base contracts for your plugins, we do offer some options for you to inherit from and speed up development.
 
-Depending on the use-case of your plugin, you may need it to be:
+The needs of your plugin determine the type of plugin you may want to choose. This is based on:
 
-- upgradeable or non-upgradeable
-- deployed by a specific deployment method
-- compatible with meta transactions
+- the need for a plugin's upgradeability
+- whether you need it deployed by a specific deployment method
+- whether you need it to be compatible with meta transactions
```

Comment:
This change adds redundancy/repetition IMO

Reply:
It's a TLDR at the beginning of the doc, assuming devs will mostly skim through documentation. I'd rather keep it even if it repeats a bit of what will be said underneath.